hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineProvider.java

```diff
@@ -100,7 +100,7 @@ List<DatanodeDetails> pickNodesNotUsed(REPLICATION_CONFIG replicationConfig,
               "data in healthy node set. Nodes required: %d Found: %d",
           metadataSizeRequired, dataSizeRequired, nodesRequired,
           healthyDNsWithSpace.size());
-      LOG.error(msg);
+      LOG.warn(msg);
```

Comment:
`pickNodesNotUsed` seems to be used only for RATIS/ONE pipeline.  Similar message is logged as error for RATIS/THREE pipeline elsewhere:

https://github.com/apache/ozone/blob/74de0705c916bb877eae44310f43a0d7069cdef9/hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/SCMCommonPlacementPolicy.java#L280-L288

Can you please change that one, too?  (Maybe even refactor to get rid of the duplication.)